packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/container.tsx

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Just a couple thoughts on `SigninRecoveryMethod/Phone/ConfirmRecoveryCode/` path:
* If you prefer the nesting, should we at least use `/phone/` instead of `/Phone/`, since capitalization usually implies there's a component of that name?
* What's this route going to be? Can we just call it `SigninConfirmRecoveryCode` and not nest it further with `/phone/`? We do already have `Signin/SigninRecoveryCode`.